seed/challenges/03-front-end-libraries/react.json

```diff
@@ -12,7 +12,7 @@
       "src": "https://cdnjs.cloudflare.com/ajax/libs/react/15.4.1/react-dom.js"
     }
   ],
-  "template": "<body><div id='root'></div>${ source }</body>",
+  "template": "<body><div id='root'></div><div id='challenge-node'></div>${ source }</body>",
```

Comment:
This is unnecessary, I've formatted the challenges to use the root div.

Reply:
@BerkeleyTrue This is not for rendering the component's to the DOM, though it does double as this in this case. It is for the ReactDOM challenges, where campers need to actually include `ReactDOM.render()` in their code. We ask them to render to a div with id `challenge-node`. Removing the tail for those challenges prevents duplicate rendering. We could just ask them to render to `root` instead, but was hoping this would help avoid confusion and thinking they need to do this every time, since we do it in the background.

Reply:
I see, then this comment can be ignored.